scripts/hotspot.py

```diff
@@ -20,19 +21,22 @@ def parse_command():
     hotspot_options.add_argument('--genomic_hotspots',
         nargs='*',
         help="Tab-delimited BED formatted file(s) containing hg38 genomic positions corresponding to hotspots")
-    hotspot_options.add_argument('--protein_hotspots',
+    hotspot_options.add_argument('--protein_snvs',
         nargs='*',
-        help="Column-name-containing, tab-delimited file(s) containing protein names and HGVSp short values corresponding to hotspots")
+        help="Column-name-containing, tab-delimited file(s) containing protein names and position for SNV hotspots")
+    hotspot_options.add_argument('--protein_indels',
+        nargs='*',
+        help="Column-name-containing, tab-delimited file(s) containing protein names and position for INDEL hotspots")
```

Comment:
Perhaps specify here that position column should be in "range format", like `<start_aa_pos>-<end_aa_pos>`

Reply:
Sure.